spanner/spanner_snippets/snippet.go

```diff
@@ -1819,6 +1853,41 @@ func queryWithHistory(ctx context.Context, w io.Writer, client *spanner.Client)
 	}
 }
 
+// [START spanner_create_client_with_query_options]
+
+func createClientWithQueryOptions(ctx context.Context, w io.Writer, database string) error {
+	queryOptions := spanner.QueryOptions{Options: &sppb.ExecuteSqlRequest_QueryOptions{OptimizerVersion: "1"}}
+	client, err := spanner.NewClientWithConfig(
+		ctx, database, spanner.ClientConfig{QueryOptions: queryOptions},
+	)
```

Comment:
Don't think the newline is needed?

https://github.com/golang/go/wiki/CodeReviewComments#line-length

Reply:
As far as I can see, there is not line length limit in the getting started guide for Go (https://cloud.google.com/spanner/docs/getting-started/go). But this is a good point. Fixed.